regenerate_commands.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+python3 ex2_ground_station_software/src/docGen.py
```

Comment:
```suggestion
cd ex2_ground_station_software
python3 src/docGen.py
cd ..
```
I believe that docGen.py generates CommandDocs.txt in the current directory, so we'll need to go into ex2_ground_station_software first so it generates it in the correct file path.

Reply:
Oops, so it does. Looks like the suggested change works perfect.